py-polars/tests/test_df.py

```diff
@@ -1741,10 +1741,10 @@ def name_generator() -> Iterator[str]:
 
 def test_extension() -> None:
     class Foo:
-        def __init__(self, value):  # type: ignore
+        def __init__(self, value: Any) -> None:
```

Comment:
Looking at the context, `value` can be an `int` type here. And in general, `__init__` methods should not specify their return type (it is always `-> None`), unless it has no other arguments (which would make it an untyped def). Explanation [here](https://github.com/python/mypy/issues/604#issuecomment-84051143).

```suggestion
        def __init__(self, value: int):
```

Reply:
I have changed to `int`, as that is the value used later in the test. I could not drop the `None` though, mypy complains. So leaving that one in.